app/assets/javascripts/initializers/initializeArticleReactions.js

```diff
@@ -46,28 +46,6 @@ function showUserReaction(reactionName, animatedClass) {
   if (reactionDrawerButton && reactionName !== 'readinglist') {
     reactionDrawerButton.classList.add('user-activated', 'user-animated');
   }
-
-  if (animatedClass == 'user-animated') {
-    const activeIcon = reactionButton.querySelector(
-      '.crayons-reaction__icon--active img',
-    );
-
-    if (activeIcon) {
-      const activeDrawerIcon = reactionDrawerButton.querySelector(
-        '.crayons-reaction__icon--active img',
-      );
-
-      reactionDrawerButton.originalIcon = activeDrawerIcon.outerHTML;
-      activeDrawerIcon.outerHTML = activeIcon.outerHTML;
-
-      setTimeout(function () {
-        document
-          .getElementById('reaction-drawer-trigger')
-          .querySelector('.crayons-reaction__icon--active img').outerHTML =
-          reactionDrawerButton.originalIcon;
-      }, 1500);
-    }
-  }
```

Comment:
😢 to see it go - it was a fun little animation - but better to be plain than to be buggy, and I believe we'll get it sorted out eventually.